packages/site/cms-site/src/blocks/PixelImageBlock.tsx

```diff
@@ -11,14 +12,10 @@ import { PropsWithData } from "./PropsWithData";
 
 interface PixelImageBlockProps extends PropsWithData<PixelImageBlockData>, Omit<ImageProps, "src" | "width" | "height" | "alt"> {
     aspectRatio: string | "inherit";
-    layout: "fill" | "fixed" | "intrinsic" | "responsive";
-    // Workaround to prevent flicker on already loaded images.
-    // See https://github.com/vercel/next.js/issues/27539 for more information.
-    disableBlurPlaceholder?: boolean;
```

Comment:
removed this prop, it is not used in any of our projects and its fixed in next v11.2.1 (https://github.com/vercel/next.js/issues/27539#issuecomment-923618301)